Makefile.builder

```diff
@@ -1,4 +1,4 @@
-RPM_SPEC_FILES.dom0 := xen.spec
+RPM_SPEC_FILES.dom0 := $(if $(filter $(DIST_DOM0), $(DISTS_VM)),, xen.spec)
```

Comment:
There is one problem with such approach - when someone is building just dom0 packages (`make vmm-xen-dom0`, or even `make qubes-dom0`), it will build nothing.